docs/configuration/config-file-reference.md

```diff
@@ -2513,19 +2649,35 @@ api:
 
 The `configstore_config` configures the config database storing rules and alerts, and is used by the Cortex alertmanager. The supported CLI flags `<prefix>` used to reference this config block are:
 
-- `alertmanager`
-- `ruler`
+- `alertmanager.client`
+- `ruler.`
 
 &nbsp;
 
 ```yaml
-# URL of configs API server.
-# CLI flag: -<prefix>.configs.url
-[configs_api_url: <url> | default = ]
+# Endpoint to connect to
+# CLI flag: -<prefix>.http-endpoint
+[endpoint: <url> | default = ]
 
-# Timeout for requests to Weave Cloud configs service.
-# CLI flag: -<prefix>.configs.client-timeout
+# Timeout for connecting to the endpoint
+# CLI flag: -<prefix>.http-client-timeout
```

Comment:
This is a breaking change.

Also, CLI flags and YAML option should be specular. If the YAML is `client_timeout` then the CLI flag should be `client-timeout`. _Same comment applies to all other flags._